### PR TITLE
ADD: loc de, fr, pl

### PR DESCRIPTION
**Pull request by limpbrains**

### CODE REVIEW

File: ios/fastlane/metadata/fr-FR/description.txt [hunk 1]
```diff
@@ -49,5 +49,5 @@ Un portefeuille Lightning sans aucune configuration nécessaire. Des transaction
 Acheter du bitcoin
 Rejoignez la révolution la finance ouverte avec la possibilité d'acheter du bitcoin directement depuis votre portefeuille.
 
-Local Trader
+Trader Local
```
Comment (Overtorment) [inline ios/fastlane/metadata/fr-FR/description.txt:52]: btw its time we get rid of local trader texts since it got removed @ncoelho @marcosrdz

**Comment by Overtorment** [comment]
hmm. why did so many tests fail..?

File: ios/fastlane/metadata/fr-FR/description.txt (hunk 1 above)
Comment (marcosrdz) [reply ios/fastlane/metadata/fr-FR/description.txt:52]: It didnt. Its in settings - tools

**Review by marcosrdz — APPROVED** [review]

File: ios/fastlane/metadata/fr-FR/description.txt (hunk 1 above)
Comment (Overtorment) [reply ios/fastlane/metadata/fr-FR/description.txt:52]: O_o

**Pull request merged**